src/modules/email/email.controller.ts

```diff
@@ -1,35 +1,63 @@
-import { Controller, Post, Get, Body, Res } from '@nestjs/common';
+import { Controller, Post, Get, Body, Res, Patch, Param, UseGuards } from '@nestjs/common';
 import { Response } from 'express';
 import { EmailService } from './email.service';
-import { skipAuth } from '../../helpers/skipAuth';
-import { SendEmailDto, createTemplateDto, getTemplateDto } from './dto/email.dto';
+import { SendEmailDto, UpdateTemplateDto, createTemplateDto, getTemplateDto } from './dto/email.dto';
 import { skip } from 'node:test';
-import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { ApiBearerAuth, ApiOperation, ApiParam, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { SuperAdminGuard } from '../../guards/super-admin.guard';
 
-@ApiTags('Emails')
+@ApiTags('email')
```

Comment:
Api Tags should be uppercase. and can be left as Emails

Reply:
fix